sunkit_pyvista/plotter.py

```diff
@@ -170,3 +174,19 @@ def plot_field_lines(self, field_lines, **kwargs):
             field_line_mesh = pv.StructuredGrid(grid[:, 0], grid[:, 1], grid[:, 2])
             color = {0: 'black', -1: 'tab:blue', 1: 'tab:red'}.get(field_line.polarity)
             self.plotter.add_mesh(field_line_mesh, color=color, **kwargs)
+            self.meshes.append(field_line_mesh)
+
+    def save(self, filename, **kwargs):
+        """
+        Saves a given plot to a vtm file.
+
+        Parameters
+        ----------
+        filename : `str`
+            Name of the file to save as, should end with vtm.
+        """
+        all_meshes = pv.MultiBlock(self.mesh)
+        if filename.endswith('vtm'):
```

Comment:
Do we just want to tell them we will append the extension in the doc string and do away with this?

Reply:
Sure that would work too I suppose